cabal-install/src/Distribution/Client/Types/Credentials.hs

```diff
@@ -1,9 +1,15 @@
 module Distribution.Client.Types.Credentials
-  ( Username (..)
+  ( Auth
+  , Token (..)
+  , Username (..)
   , Password (..)
   ) where
 
 import Prelude (String)
+import Data.Either (Either)
```

Comment:
Is this not automatically imported from cabal prelude? And if not, why not import it in the line above?